research/conditional/utils/conditional_trainer.py

```diff
@@ -277,7 +277,7 @@ def _eval_step(self, step):
                 value=total_correct_tokens / total_masked_tokens,
                 iteration=step,
             )
-            for name, loss_value in extra_losses:
+            for name, loss_value in extra_losses.items():
                 self.logger.report_scalar(
```

Comment:
Why?

Reply:
why what? you expect a pair, so should iterate over something that returns pairs

Reply:
But this didnt work previously? Because i dont see anything related in this PR chsnging this vsriabke behaviour

Reply:
It did not. I ran the first tests with aux loss after the `eval step` feature was added.